qiskit/ignis/measurement/discriminator/discriminators.py

```diff
@@ -0,0 +1,426 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+from abc import abstractmethod
+from typing import Union, List
+
+import numpy as np
+from sklearn.discriminant_analysis import LinearDiscriminantAnalysis
+from sklearn.discriminant_analysis import QuadraticDiscriminantAnalysis
+from sklearn.preprocessing import StandardScaler
+
+from qiskit.exceptions import QiskitError
+from qiskit.pulse import PulseError
+from qiskit.result import Result
+from qiskit.pulse.schedule import Schedule
+
+
+class BaseDiscriminationFitter:
```

Comment:
Maybe this should be in something like `fitter.py`, or the implemented discriminator should be moved to a new module?

Reply:
This is something I wanted to get to. How about having `BaseDiscriminationFitter` in `fitters.py` and `IQDiscriminationFitter` along with the linear and quadratic implementation in `iq_discrimination.py`?